InventoryWatch/InventoryWatchApp.swift

```diff
@@ -8,6 +8,68 @@
 import SwiftUI
 
 @main
+struct AppVersionSelector {
+    static func main() {
+        if #available(macOS 13.0, *) {
+            InventoryWatchAppMacOS13.main()
+        } else {
+            InventoryWatchApp.main()
+        }
```

Comment:
tbh I'd love _not_ to do this, but after poking at it for a bit, I'm not sure there's another way... it sucks to have two nearly identical `App` structs, but since `SceneBuilders` don't support branching logic, idk if there's a better way. perhaps the `WindowGroup` and `Settings` code could be encapsulated into separate functions that are included in both apps?

Reply:
Yeah, I tried that earlier and thought then that the readability wasn't great, but I've made the changes now and think it looks fine.